crates/core/src/client/mod.rs

```diff
@@ -559,7 +556,7 @@ impl<P: Provider + Send + Sync> KakarotEthApi<P> for KakarotClient<P> {
         let data = request.data.unwrap_or_default();
 
         let tx = Transaction::Eip1559(TxEip1559 {
-            chain_id,
+            chain_id: chain_id.low_u64(),
```

Comment:
how come request.chain_id is not the same format as TxEip1559.chain_id?

Reply:
request is U64 and TxEip1559 is u64, not sure why